index.html

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
+<html>
+<head>
+<link rel="stylesheet" type="text/css" href="style.css">
+<link rel="stylesheet" href="css/font-awesome.min.css">
+<title>Tim's Portfolio</title>
+</head>
+<body>
+  <div id="nav-bar" class="nav-bar">
+      <h1 id="home-nav">Home</h1>
+      <h1 id="work-nav">Projects</h1>
+      <h1 id="about-nav">About Me</h1>
+    </div>
```

Comment:
Avoid using divs when possible.  This should be a ul with li's all inside a nav.